src/Api.Rest.Dtos/PathHelper.cs

```diff
@@ -153,139 +158,196 @@ public static PathInformationDto RoundtripString2PathInformation( [NotNull] stri
 				throw new ArgumentException( "The path string must not be null or empty.", nameof( path ) );

```

Comment:
RoundtripString2PathInformation seems to be missing a check or usefull exception when the structure does not correspond with the actual path, e.g. "PP:part1/part2/part3". There was an exception before (The number of components in path string '{path}' and structure '{maybeStructure}' must be equal.)

Reply:
Maybe you could also add some test for this to the PathHelperTests.

Reply:
Fixed. Thanks for the perfect help by writing tests.